src/routes/commentRouter.js

```diff
@@ -0,0 +1,24 @@
+import express from 'express';
+import {addComment, deleteComment, getComments} from '../controllers/commentController';
+import {errorHandler, isLoggedIn, validate}  from '../middlewares';
+import { validateComment } from '../validations/commentValidation';
+
+const commentRouter = express.Router();
+
+commentRouter.post(
+  '/trip/:tripId/comment',
+  validate(validateComment),
+  isLoggedIn,
+  errorHandler(addComment)
+);
```

Comment:
Someone who is commenting should Have made request on that particular trip. How do you handle that?